src/http_downloader.js

```diff
@@ -52,34 +52,36 @@ class HttpDownloader extends EventEmitter {
 		}
 	}
 
-	downloadResource(resource) {
+	downloadResource(name, resource) {
 		const url = new URL(resource['url']);
-		const name = resource['destination'];
-		const destination = path.join(springPlatform.writePath, name);
+		this.name = name;
```

Comment:
Why change it from local variable that is then just referenced from closure to object property?